site/docs/samples/notebook.md

```diff
@@ -0,0 +1,219 @@
+    
+    1. Install Minio to the currently active namespace:
+      ```bash
+      helm repo add minio https://helm.min.io
+      helm install --wait minio minio/minio
+      ```
+    1. Create a port-forward to connect to MinIO UI:
+      ```bash
+      kubectl port-forward svc/minio 9000 &
+      ```
+    1. Open [http://localhost:9000](http://localhost:9000) and login with the printed access key and secret key:
+      ```
+      ACCESS_KEY=$(kubectl get secret minio -o jsonpath="{.data.accesskey}" | base64 --decode)
+      SECRET_KEY=$(kubectl get secret minio -o jsonpath="{.data.secretkey}" | base64 --decode)
+      echo "Access Key: ${ACCESS_KEY}"
+      echo "Secret Key: ${SECRET_KEY}"
+      ```
+    1. Click the :fontawesome-solid-plus-circle: button in the bottom right corner and then **Create bucket** to create a bucket (e.g. "demo").
+    1. Click the :fontawesome-solid-plus-circle: button again and then **Upload files** to upload the CSV file to the newly created bucket.
+    
+## Register the dataset in a data catalog
+
+Register the credentials required for accessing the dataset. Replace the values for `accessKey` and `secretKey` with the values from the object storage service that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: data-csv
+type: Opaque
+stringData:
+  accessKey: "${ACCESS_KEY}"
+  secretKey: "${SECRET_KEY}"
+EOF
+```
+
+Then, register the data asset itself in the catalog. Replace the values for `endpoint`, `bucket` and `objectKey` with values from the object storage service that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.m4d.ibm.com/v1alpha1
+kind: Asset
+metadata:
+  name: data-csv
+spec:
+  secretRef: 
+    name: data-csv
+  assetDetails:
+    dataFormat: csv
+    connection:
+      type: s3
+      s3:
+        endpoint: "http://minio:9000"
+        bucket: "demo"
+        objectKey: "data.csv"
+  assetMetadata:
+    geography: us-south
+    tags:
+    - finance
+    componentsMetadata:
+      nameOrig: 
+        tags:
+        - PII
+      oldbalanceOrg:
+        tags:
+        - sensitive
+      newbalanceOrig:
+        tags:
+        - sensitive
+EOF
+```
+
+The asset is now registered in the catalog. The identifier of the asset is `m4d-notebook-sample/data-csv` (i.e. `<namespace>/<name>`). You will use that name in the `M4DApplication` later.
+
+Notice the `assetMetadata` field above. It specifies the dataset geography and tags. These attributes are later used in policies.
+
+
+## Define data access policies
+
+Currently predefined policies are included as part of the OPA deployment.
+Included are policies that are triggered for datasets that are tagged with 'finance' and have columns `nameOrig` and `nameDest`. The policies indicate that these columns must be redacted (masked) when data is read.
+
+The policies can be found at `third_party/opa/opa-policy.rego`.
```

Comment:
the opa-policy.rego is being removed through the PR https://github.com/IBM/the-mesh-for-data/pull/442. Do you suggest that we create a new folder for opa-policy.rego and add this file  as we have done [here](https://github.com/rohithdv/the-mesh-for-data/tree/opa_use_kube_mgmt3/third_party/opa/data-and-policies/user-created-policy-1) ?

Reply:
Please add a suggestion for a policy to be applied directly by the user post #442 to get the same effect that is demonstrated in this sample.